util/lru/lru.go

```diff
@@ -22,14 +18,22 @@ type Cache[K comparable, V any] struct {
 	// an item is evicted. Zero means no limit.
 	MaxEntries int
 
-	ll *list.List
-	m  map[K]*list.Element // of *entry[K,V]
+	// l is a ring of LRU values. l points to the most recently used
```

Comment:
I try to avoid `l` as a variable name because in many fonts it's hard to distinguish between `1`, `l`, and `I`.

Maybe `r` for ring?

Reply:
or `h` for head?

Reply:
Renamed to `c.head`, and `c.m` to `c.lookup`. While really short var names are nice, in this case the savings on typing seem less important than readability.